src/lib/saml.ts

```diff
@@ -119,7 +119,7 @@ export const configSaml = async (passport: PassportWithLogout) => {
       } else {
         // DEVELOPMENT ONLY
         // The test IDP only supports nameIDFormat=transient, so we need to use a different attribute
-        userId = samlUser.attributes.userprincipalname
+        userId = samlUser.attributes.common_name
```

Comment:
changing to `common_name` to bring our test code in alignment with prod

Reply:
Thank you! This will avoid some future confusion.